core/trino-main/src/main/java/io/trino/operator/scalar/DateTimeFunctions.java

```diff
@@ -85,6 +88,8 @@
     private static final int MILLISECONDS_IN_HOUR = 60 * MILLISECONDS_IN_MINUTE;
     private static final int MILLISECONDS_IN_DAY = 24 * MILLISECONDS_IN_HOUR;
     private static final int PIVOT_YEAR = 2020; // yy = 70 will correspond to 1970 but 69 to 2069
+    private static final Slice YmD = Slices.utf8Slice("%Y-%m-%d");
```

Comment:
would it be correct to call it `iso8861DateFormat`?

Reply:
update in d6f05bd8ec6e41ee32e70f8d6cfb128e447d74b0